src/components/Connections/connectionTypes.tsx

```diff
@@ -1807,6 +1807,55 @@ export const connectionTypes: ConnectionType[] = [
         }
       };
     }
+  },
+  {
+    title: "Git",
+    icon: "git",
+    value: ConnectionValueType.Git,
+    fields: [
+      {
+        label: "Name",
+        key: "name",
+        type: fieldTypes.input,
+        required: true
+      },
+      {
+        label: "Certificate",
```

Comment:
We need a username and password as well and rename the certificate label to SSH Key
+ reorder name, url, user, pass, certificate, ref